CONTRIBUTING.md

```diff
@@ -16,7 +16,7 @@ Please see the [Developer Contributions section](/docs/contributors/develop.md)
 
 ## How Can Designers Contribute?
 
-If you'd like to contribute to the design or front-end, feel free to contribute to tickets labelled [Needs Design](https://github.com/WordPress/gutenberg/issues?q=is%3Aissue+is%3Aopen+label%3A%22Needs+Design%22) or [Needs Design Feedback](https://github.com/WordPress/gutenberg/issues?q=is%3Aissue+is%3Aopen+label%3A"Needs+Design+Feedback%22). We could use your thoughtful replies, mockups, animatics, sketches, doodles. Proposed changes are best done as minimal and specific iterations on the work that precedes it so we can compare. If you use <a href="https://www.sketchapp.com/">Sketch</a>, you can grab <a href="https://cloudup.com/cMPXM8Va2cy">the source file for the mockups</a> (updated April 6th).
+If you'd like to contribute to the design or front-end, feel free to contribute to tickets labelled [Needs Design](https://github.com/WordPress/gutenberg/issues?q=is%3Aissue+is%3Aopen+label%3A%22Needs+Design%22) or [Needs Design Feedback](https://github.com/WordPress/gutenberg/issues?q=is%3Aissue+is%3Aopen+label%3A"Needs+Design+Feedback%22). We could use your thoughtful replies, mockups, animatics, sketches, doodles. Proposed changes are best done as minimal and specific iterations on the work that precedes it so we can compare. The [WordPress.org Design team](http://make.wordpress.org/design/) uses [Figma](https://www.figma.com/). If you'd like to contribute, consider [joining the The WordPress.org Figma team](https://make.wordpress.org/design/2018/11/19/figma-for-wordpress/). The Figma team has access to a helpful [library of components](https://www.figma.com/file/ZtN5xslEVYgzU7Dd5CxgGZwq/WordPress-Components?node-id=0%3A1) used in WordPress.  
```

Comment:
A few small suggested changes here: 

- "WordPress.org Design team" → "WordPress design team"
- "uses Figma" → "uses Figma to more easily collaborate and share work" (or similar)
- "consider joining The WordPress.org Figma team" → "ping in Slack and the design team will set you up with a free Figma account" (the process is a bit different and basically you need to ping an admin)

Reply:
Thanks @sarahmonster! Updated in 5ad4184.

Reply:
💯 that reads a million times better. 🙌 🚢